evm/src/witness/traces.rs

```diff
@@ -48,6 +49,47 @@ impl<T: Copy> Traces<T> {
         }
     }
 
+    /// Returns the actual trace lengths for each STARK module.
+    //  Uses a `TraceCheckPoint` as return object for convenience.
+    pub fn get_lengths(&self) -> TraceCheckpoint {
+        TraceCheckpoint {
+            arithmetic_len: self
+                .arithmetic_ops
+                .iter()
+                .map(|op| match op {
+                    Operation::TernaryOperation {
+                        operator: _,
```

Comment:
I think these unused struct fields in the match can be replaced with `..`

Reply:
Learning new things every day 😄 Thanks! Definitely much cleaner that way!